apps/api/src/server/repositories/representation.repository.js

```diff
@@ -53,6 +53,92 @@ export const getByCaseId = async (caseId, { page, pageSize }, { searchTerm, filt
 	};
 };
 
+/**
+ *
+ * @param {number} id
+ * @param {number?} caseId
+ * @returns {Promise<any>}
+ */
+export const getById = async (id, caseId) => {
+	let caseFilter = {};
+
+	if (caseId) {
+		caseFilter = {
+			case: {
+				id: caseId
+			}
+		};
+	}
+
+	const representations = await databaseConnector.representation.findMany({
+		select: {
+			id: true,
+			reference: true,
+			status: true,
+			redacted: true,
+			received: true,
+			originalRepresentation: true,
+			redactedRepresentation: true,
+			user: {
+				select: {
+					azureReference: true
+				}
+			},
+			contacts: {
+				select: {
+					type: true,
+					firstName: true,
+					lastName: true,
+					organisationName: true,
+					jobTitle: true,
+					under18: true,
+					email: true,
+					phoneNumber: true,
+					address: {
+						select: {
+							addressLine1: true,
+							addressLine2: true,
+							town: true,
+							county: true,
+							postcode: true
+						}
+					}
+				}
+			},
+			attachments: {
+				select: {
+					documentVersion: {
+						select: {
+							fileName: true,
+							mime: true,
+							path: true
+						},
+						take: 1,
+						orderBy: {
+							lastModified: 'desc'
+						}
+					}
+				},
+				where: {
+					isDeleted: false
+				}
+			}
+		},
+		where: {
+			id,
+			...caseFilter
+		}
+	});
+
+	const representation = representations[0];
+
+	if (representation === null || typeof representation === 'undefined') {
+		throw new Error('Unknown representation on application');
+	}
```

Comment:
I would say we need to remove this logic out of this function. A few reasons for this:
1. The validator defined in `application.validators.js` will never be invoked
2. For different endpoints, we might want to handle the missing representations differently (some might want to throw and error, some might want to swallow the error). 
3. We generally do not perform these checks within any other repository and it is therefore a little confusing to have differing behaviour for this repository.

Reply:
So the repository should returned the raw result ? Which could be undefined / null ? 

And then each calling function (such as getCaseRepresentation) would need to handle the response.

Reply:
That is what I would lean towards yes

Reply:
i will address later today.